### DISCUSSION TITLE
Missing values in filter JobTitle Sharepoint Search Persons template

### DISCUSSION THREAD

**Discussion by Wernbe** [microsoft-search/pnp-modern-search#2315] [Q&A] [answered]
Dear PnP Modern Search team,

Great module you build! We use it quit often. For an client we want to use it as a people finder. We use version 4.6.1.0. When using the search filters webpart not all the unique jobtitles are in the list. Totaal amount of people is around 890 in the results list. The person with the missing jobtitle is in de result list.

Hopefully somebody know how to show all the unique jobtitles in the filter of the result.

**Comment by wobba** [comment, marked as answer]
Enter a high value in "Max # of values" in the filter settings. https://microsoft-search.github.io/pnp-modern-search/usage/search-filters/#filter-settings
